pombola/south_africa/tests.py

```diff
@@ -684,7 +684,7 @@ def test_attendance_data(self):
 
         self.assertEqual(
             context['attendance'],
-            [{'total': 28, 'percentage': 89.28571428571429, 'attended': 25, 'year': 2015, 'position':'mp'},
```

Comment:
There is one alternate_member attendance in the [data](https://github.com/OpenUpSA/pombola/blob/master/pombola/south_africa/data/test/attendance_587.json) that this test uses That's why each of the present counts goes down by 1.